### ISSUE TITLE
Standard Sample Pattern

### ISSUE THREAD

**Issue by kainino0x** [gpuweb/gpuweb#4005]
I tightened some of our existing tests to test the exact pattern of multisampling samples: https://github.com/gpuweb/cts/pull/2466

- D3D11 requires standard patterns, and D3D12 should by extension.
  https://learn.microsoft.com/en-us/windows/win32/api/d3d11/ne-d3d11-d3d11_standard_multisample_quality_levels
- Vulkan has the [standardSampleLocations](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkPhysicalDeviceLimits.html#limits-standardSampleLocations) option, which is supported universally.
- Metal documentation specifies that sample locations are not standardized on devices that don't have programmable sample locations: https://developer.apple.com/documentation/metal/mtldevice/2866120-getdefaultsamplepositions#discussion
   However, all devices in the current [Metal Feature Set Tables](https://developer.apple.com/metal/Metal-Feature-Set-Tables.pdf) support programmable sample locations! According to [archived copies of that document](https://web.archive.org/web/20230000000000*/https://developer.apple.com/metal/Metal-Feature-Set-Tables.pdf), it was only ever unsupported on iOS before 11, and on macOS before 10.13. We require both, so we're in the clear.
   Also, my tests pass on M1 on macOS 13.2.1. All other desktop Mac hardware supported D3D11 so would also have the standard sample pattern anyway.

In conclusion, I think we can finally formally decide we are using the standard sample pattern.
